protocol/flows/src/service.rs

```diff
@@ -71,8 +75,11 @@ impl AsyncService for P2pService {
 
         // Launch the service and wait for a shutdown signal
         Box::pin(async move {
-            if let Some(peer_address) = self.connect.clone() {
-                connection_manager.add_connection_request(peer_address.to_socket_addrs().unwrap().next().unwrap(), true).await;
+            for peer_address in self.connect_peers.iter().cloned() {
```

Comment:
I think in go-kaspad having `connect` means exclusiveness. Meaning the node does not attempt any other outbound connections (can be achieved by setting `outbound_target=0` if `!self.connect_peers.is_empty()`. I'm not sure about inbound connections

Reply:
Right, this exclusiveness is achieved in go-kaspad by setting outbound target to 0 and disabling DNS seeding. I added that.

I could find no impact of the presence of `connect` in go-kaspad on inbound connections.

I also discovered and fixed another little discrepancy: added peers have permanent connections.